webapp/stores/channel_store.jsx

```diff
@@ -19,7 +19,7 @@ class ChannelStoreClass extends EventEmitter {
     constructor(props) {
         super(props);
 
-        this.setMaxListeners(15);
+        this.setMaxListeners(600);
```

Comment:
Yeah, it's listened to by every post component for rendering `!channel` mentions so I reused my rough estimate of 5 post list components * 100 posts